docs/pricing/am2-pricing.mdx

```diff
@@ -0,0 +1,262 @@
+---
+title: Pricing & Billing (Before June 11, 2024)
+sidebar_hidden: true
+sidebar_order: 1
+description: "Learn about pricing variables, managing volume, and different Sentry plans."
+---
+
+<Alert>
+This pricing only applies to paid plans that started **before** June 11, 2024 and have not upgraded to the current pricing plan.
+
+If you're on a free or paid plan that started on June 11, 2024 or later, see the [current pricing page](/pricing/).
```

Comment:
```suggestion
This information only applies to paid plans that were activated **before June 11, 2024** and haven't been upgraded since.

If you're on a free or paid plan that was activated on June 11, 2024 or later, see the [current pricing page](/pricing/).
```